app/services/age_authorization_handler.rb

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
Is it missing the `# frozen_string_literal: true` comment?

Reply:
Good catch! I guess I accidentally removed it 🤦🏻